### PR TITLE
wire up some meta client funcs and tests

### PR DESCRIPTION
**Pull request by dgnorton**

### CODE REVIEW

File: services/meta/service_test.go [hunk 1]
```diff
@@ -205,3 +477,75 @@ func mustParseStatement(s string) influxql.Statement {
 	}
 	return stmt
 }
+
+// Test long poll of snapshot.
```
Comment (pauldix) [inline services/meta/service_test.go]: can you remove all this commented stuff? The long poll is already tested on every call the client makes that creates something.

**Comment by pauldix** [comment]
Looks good, just a thing to remove commented code. You don't want to make the proto change in this one?

File: services/meta/service_test.go (hunk 1 above)
Comment (dgnorton) [inline services/meta/service_test.go]: ah...yeah, I had removed it originally.  That just didn't survive the rebase.  :)  I'll remove it.

**Comment by dgnorton** [comment]
@pauldix unless it will mess something up, I'd say let's merge this much and I'll make the proto change with the next batch of changes

**Comment by pauldix** [comment]
sounds good to me

**Pull request merged**